crates/pevm/src/chain.rs

```diff
@@ -50,10 +51,10 @@ pub trait PevmChain: Debug {
     type Envelope: Debug + From<Signed<TxLegacy>>;
 
     /// The error type for [`Self::get_block_spec`].
-    type BlockSpecError: Debug + Clone + PartialEq;
+    type BlockSpecError: StdError + Debug + Clone + PartialEq + 'static;
 
     /// The error type for [`Self::get_tx_env`].
-    type TransactionParsingError: Debug + Clone + PartialEq;
+    type TransactionParsingError: StdError + Debug + Clone + PartialEq + 'static;
```

Comment:
nit, can we standardize the order of traits?
- `Debug`
- `Clone`, `Copy`
- `PartialEq`, `Eq`
- `StdError`, `thiserror::Error`
- `'lifetime`

Reply:
`thiserror::Error` is not a trait :upside_down_face:. It just derives `std::error::Error + Display`.

I don't have a strong preference, but less common/more complex traits usually go to left, while trait markers and lifetime is rightmost. So IDK, this ordering makes sense, as Debug/Clone/Copy/PartialEq/Eq are rust builtins

Reply:
oh ok, your explanation makes sense